packages/cli/lib/resources/template.html

```diff
@@ -10,8 +10,9 @@
 		<% if (htmlWebpackPlugin.options.manifest.theme_color) { %>
 			<meta name="theme-color" content="<%= htmlWebpackPlugin.options.manifest.theme_color %>">
 		<% } %>
+		<% const filesRegexp = options.criticalCss ? /\.(js)$/ : /\.(css|js)$/;%>
```

Comment:
I don't understand – we'd still want to preload _some_ CSS, especially the routes' chunks.